app/src/main/res/layout/music_nowplaying.xml

```diff
@@ -78,7 +78,7 @@
                     android:scaleType="centerCrop"
                     android:src="@drawable/ic_error_outline_black_24dp"
                     app:tint="?android:attr/textColorSecondary"
-                    android:visibility="@{viewModel.errorTitle != null}"/>
+                    android:visibility="@{viewModel.errorTitle != null || viewModel.isWebApiAuthorized != null}"/>
```

Comment:
The viewModel's errorTitle was quickly ported over from the previous version's error CLASSNAME. Feel free to reuse the errorTitle field for a message to request Web API authentication, and then this layout doesn't need this extra logic.

Reply:
One problem with using the errorTitle field for the authentication message is that if the user chooses not to use the web API and is not authenticated (or just doesn't want to at the time for whatever reason) and an app remote error comes up the web api authorization error will always take the error title/message place and thus block out any incoming errors from that. The way I have the SpotifyErrorApiDialog set up is so that the app remote errors and the web api errors can be shown side by side that way nothing is lost out

Reply:
Here's an example of this behavior for reference: 
![Screenshot_20201223-204258_AndroidAutoIdrive](https://user-images.githubusercontent.com/11298573/103054377-93464e00-455f-11eb-8600-aa6e96d100de.jpg)